### ISSUE TITLE
DropboxOAuth2Helper.ParseTokenFragment call cause an ArgumentException

### ISSUE THREAD

**Issue by yuriy-blinov** [dropbox/dropbox-sdk-dotnet#26]
Recently  DropboxOAuth2Helper.ParseTokenFragment started to throw ArgumentException during  authentication.

```
case WebAuthenticationStatus.Success:
					var response = DropboxOAuth2Helper.ParseTokenFragment(new Uri(result.ResponseData));
```
ResposeData= ms-app://s-x-xx-x-xxxxxxxxxx-xxxxxxxxx-xxxxxxxxxx-xxxxxxxxxx-xxxxxxxxxx-xxxxxxxxx-xxxxxxxxxx/#access_token=xxyvxxxxxxgxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxwx-xxxxotxtkxxxml_f&token_type=bearer&uid=xxxxxxxx&account_id=dbid%xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Probably the issue happens because **account_id** is present.

```
[System.ArgumentException] = {System.ArgumentException: Unexpected values in fragment
Parameter name: redirectedUri
   at Dropbox.Api.DropboxOAuth2Helper.ParseTokenFragment(Uri redirectedUri)
....
```

**Comment by greg-db** [comment]
Thanks for the report! We're looking into it.

**Comment by greg-db** [comment]
This should be fixed in [3.2.1](https://github.com/dropbox/dropbox-sdk-dotnet/releases/tag/v3.2.1). Try that and let us know if you're still seeing any issues.
